src/platform_impl/macos/util/async.rs

```diff
@@ -182,7 +184,7 @@ pub unsafe fn set_maximized_async(
         } else {
           shared_state_lock.saved_standard_frame()
         };
-        ns_window.setFrame_display_(new_rect, NO);
+        let _: () = msg_send![*ns_window, setFrame:new_rect display:NO animate: YES];
```

Comment:
`ns_window.setFram_display_` is not supported animation so we need to use `ns_window:setFrame` directly.